src/test/integration/integration_tests.py

```diff
@@ -61,19 +62,35 @@ def test_missing_param(self, param, valid_params):
 
 def set_course_data(subject):
     terms = utils.get_courses(None)
-    term_id = terms.json()[0]["id"]
-    (valid_year, valid_term) = term_id.split("-")
-    courses = utils.get_courses({"term_id": term_id, "id": subject})
-    valid_course = courses.json()[0]["id"]
-    valid_section = courses.json()[0]["sections"][0]["name"]
-    global book_params
-    book_params = {
-        "academicYear": valid_year,
-        "term": valid_term,
-        "subject": subject,
-        "courseNumber": valid_course,
-        "section": valid_section
-    }
+    # Nested for loop to loop through each term, and each course, to return
+    # the first valid term, course, and a random section ther is
+    for term in terms.json():
+        term_id = term["id"]
+        (valid_year, valid_term) = term_id.split("-")
+        courses = utils.get_courses({"term_id": term_id, "id": subject})
+        # looping through each course in the term.
+        for course in courses.json():
+            valid_course = course["id"]
+            # if sections is empty loop agian to find a course
+            # with a valid section
+            if course["sections"]:
+                # include a random section in the response
+                random_section = random.randint(0, len(course["sections"]) - 1)
```

Comment:
That could work. Just wanna let you know you can also use [random.sample](https://docs.python.org/3.7/library/random.html#random.sample) to pick samples directly.